coffee/vex.coffee

```diff
@@ -150,7 +150,19 @@ vexFactory = ($) ->
                 $('body').trigger 'vexAfterClose', options # Triggered on the body since $vexContent was removed
                 options.afterClose $vexContent, options if options.afterClose
 
-            if animationEndSupport
+            hasAnimation = ->
+                $clone = $ $vex[0].cloneNode false
```

Comment:
This is the only step that mildly concerns me. My initial reaction is do we really need to clone the node and all of its contents? Perhaps all we need to do is create a test element with the same inline attributes (classes, id, data attributes, inline styles, etc.) and make it a sibling.